src/test_typing.py

```diff
@@ -5,7 +5,7 @@
 import sys
 from unittest import TestCase, main, skipUnless, SkipTest
 if sys.version_info[:2] >= (3, 6):
-    import ann_module, ann_module2, ann_module3
+    from test import ann_module, ann_module2, ann_module3
```

Comment:
Honestly I think this points to a different problem.

The stdlib tests for variable annotations syntax should be moved to a different file altogether (maybe test_annotations.py?). Then we won't need this stuff here. (Also, I really want the text of typing.py and test_typing.py to be identical in Py3.5, Py3.6, and here.)